### ISSUE TITLE
The mean and std should be calculated against the training data?

### ISSUE THREAD

**Issue by ChicoQ** [alipay/cvpr2020-plant-pathology#1]
Hi there,

The default mean and std should be changed according the training data before feeding it into the Normalize() over here
https://github.com/alipay/cvpr2020-plant-pathology/blob/master/dataset.py#L101 I'm afraid.

**Comment by nickyi1990** [comment]
we used mean and std from pretrained model.

**Comment by nickyi1990** [comment]
> according

If u don't have any problem, could u mind me close this issue?

**Comment by ChicoQ** [comment]
Thanks Nick.
